src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -222,6 +222,10 @@ public static void Clear()
         {
             if (!Console.IsOutputRedirected)
             {
+                if (!string.IsNullOrEmpty(TerminalFormatStringsInstance.ClearScrollbackBuffer))
+                {
+                    WriteStdoutAnsiString(TerminalFormatStringsInstance.ClearScrollbackBuffer);
+                }
                 WriteStdoutAnsiString(TerminalFormatStringsInstance.Clear);
```

Comment:
This is the only place Clear is used, right?  Rather than doing two separate writes, could we just update TerminalFormatStringsInstance.Clear to be constructed as the concatenation with ClearScrollbackBuffer if it exists, and then this call site doesn't change from what it was before... it just writes out Clear, which will issue both commands if they're supported.